### PR TITLE
Rename top nav blank state to "No data source"

### PR DESCRIPTION
**Pull request by amacneil**
**User-Facing Changes**
N/A (feature flagged)

**Description**
@2metres

### CODE REVIEW

File: packages/studio-base/src/i18n/ja/appBar.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const appBar: Partial<TypeOptions["resources"]["appBar"]> = {
   about: "情報",
   openLocalFile: "ローカルファイルを開く",
   openConnection: "接続を開く",
-  openDataSource: "データソースを開く…",
+  noDataSource: "データソースなし",
```
Comment (jtbandes) [inline packages/studio-base/src/i18n/ja/appBar.ts:11]:
```suggestion
```

File: packages/studio-base/src/i18n/zh/appBar.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { TypeOptions } from "i18next";
 export const appBar: Partial<TypeOptions["resources"]["appBar"]> = {
   about: "关于",
   openConnection: "打开连接……",
-  openDataSource: "打开数据源……",
+  noDataSource: "无数据源",
```
Comment (jtbandes) [inline packages/studio-base/src/i18n/zh/appBar.ts:10]:
```suggestion
```

Comment (amacneil) [reply packages/studio-base/src/i18n/zh/appBar.ts:10]: image

Comment (jtbandes) [reply packages/studio-base/src/i18n/zh/appBar.ts:10]: And did you confirm with a native speaker that those are correct?

Comment (amacneil) [reply packages/studio-base/src/i18n/zh/appBar.ts:10]: wait why did you propose deleting it?

Comment (jtbandes) [reply packages/studio-base/src/i18n/zh/appBar.ts:10]: Just following the [contributing guidelines](https://github.com/foxglove/studio/blob/main/CONTRIBUTING.md#add-translations):

> We value having _high-quality_ translations over having _all_ translations…

Comment (amacneil) [reply packages/studio-base/src/i18n/zh/appBar.ts:10]: ok up to you. I had a decent chat with gpt about it but have not asked a native speaker. if you prefer to delete them that's fine too

**Review by jtbandes — APPROVED** [review]

**Pull request merged**